### PR TITLE
feat: made self-introduction page on intro.html

### PR DESCRIPTION
**Pull request by dozukwang**
- charset, title, author, page description is on head
- informations of author is on body(name, age, from)

### CODE REVIEW

File: intro.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+
+<html>
+
+ <head>
+ <meta charset="utf-8">
+ <meta name="author" content="Dahyun Kang">
```
Comment (7louisjeon) [inline intro.html:7]: Make the last name come first pls.

File: intro.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ <h1> nice to meet you</h1>
+ <div class="list">
+  <div> name </div>
+  <div> age </div>
+  <div> from </div>
+ </div>
+
+ <div class="answer">
+  <div> Dahyun Kang </div>
+  <div> Not found </div>
```
Comment (7louisjeon) [inline intro.html:22]: Let's set the age value to 25.

File: intro.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<body>
+ <h1> nice to meet you</h1>
+ <div class="list">
+  <div> name </div>
+  <div> age </div>
+  <div> from </div>
+ </div>
+
+ <div class="answer">
+  <div> Dahyun Kang </div>
```
Comment (7louisjeon) [inline intro.html:21]: Change the location of first and last name here as well.

**Review by 7louisjeon — CHANGES_REQUESTED** [review]
Some changes should be made.

File: intro.html (hunk 1 above)
Comment (dozukwang) [reply intro.html]: I made changes.
please check this out.

File: intro.html (hunk 2 above)
Comment (dozukwang) [reply intro.html]: I made changes.
please check this out.

File: intro.html (hunk 3 above)
Comment (dozukwang) [reply intro.html]: I made changes.
please check this out.

**Comment by 7louisjeon** [comment]
Looks good to me!

**Review by 7louisjeon — APPROVED** [review]
Well done!

**Pull request merged**